packages/components/organisms/f-checkout/src/services/tests/addressGeocodingApi.test.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import addressGeocodingApi from '../addressGeocodingApi';
+import { mockAuthToken } from '../../components/_tests/helpers/setup';
+
+const locationData = {
+    addressLines: [
+        '1 Jazz Avenue',
+        'Strange Town',
+        'JZ1 1AA'
+    ]
+};
+
+const authToken = mockAuthToken;
+
+describe('AddressGeocodingApi', () => {
+    let payload;
+    let state;
+    let config;
+
+    beforeEach(() => {
+        // Arrange
+        payload = {
+            url: 'http://localhost/account/checkout',
+            timeout: 10000,
+            postData: locationData
+        };
+        config = {
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${authToken}`
+            },
+            timeout: payload.timeout
+        };
+        state = {
+            authToken
+        };
+
+        axios.post = jest.fn(() => Promise.resolve({ status: 200 }));
+    });
+    describe('getAvailableFulfilment ::', () => {
```

Comment:
Nit: add an empty line in between blocks, please.

Reply:
The `describe` seems incorrect, since we're testing `getGeoLocation`, not `getAvailableFulfilment`.